app/src/main/java/org/oppia/app/profile/AddProfileActivity.kt

```diff
@@ -11,6 +12,14 @@ class AddProfileActivity : InjectableAppCompatActivity() {
   @Inject
   lateinit var addProfileFragmentPresenter: AddProfileActivityPresenter
 
+  companion object {
+    fun createAddProfileActivityIntent(context: Context, color: Int): Intent {
```

Comment:
Is ``color`` a resource ID, a hex value, or something else? Please update the name to be specific. Ditto for the ``KEY_PROFILE_AVATAR_COLOR``.